### PR TITLE
Implement env-and-seed-setup.

### PR DESCRIPTION
**Pull request by stevekriz**
- Implement database and database seeder (using faker).
- Implement server and connect to database.
- Implement get reviews route.
- Connect back end to front end.

### CODE REVIEW

File: server/controllers/ReviewsController.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const ReviewsModel = require('../models/ReviewsModel.js');
+
+const getReviews = (req, res) => {
+  ReviewsModel.getReviews(req, (err, success) => {
```
Comment (oransproduce) [inline server/controllers/ReviewsController.js:4]: I would only pass the necessary information to the model.getReviews - the main downside given my limited understanding of hardware constraints is that it'll require a lot more memory as the number of requests gets higher. Obviously out of the scope of this project but it's always nice to keep efficiency in mind

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const path = require('path');
+const bodyParser = require('body-parser');
+const routes = require('./routes.js');
+
+const app = express();
+const port = 3000;
+
+app.use(bodyParser.urlencoded({ extended: false }));
```
Comment (oransproduce) [inline server/index.js:9]: I always do extended: true, but not sure if it makes a difference.

File: server/routes.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const ReviewsController = require('./controllers/ReviewsController.js');
+
+const router = express.Router();
```
Comment (oransproduce) [inline server/routes.js:4]: Impressed by using a router

File: src/components/Reviews.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  const [reviews, setReviews] = useState([]);
+
+  const getReviews = () => {
+    axios.get('/api/reviews/1')
+      .then(({ data }) => setReviews(data))
+      .catch((err) => console.log(err));
+  };
+
+  useEffect(() => {
+    getReviews();
```
Comment (oransproduce) [inline src/components/Reviews.jsx:14]: love the hooks

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  mode: 'development',
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: ['@babel/preset-env'],
```
Comment (oransproduce) [inline webpack.config.js:14]: I set mine up putting both presets into the array and not configuring babel as we discussed this is an option for webpack 5 but not for 4 but something to note.

**Review by oransproduce — COMMENTED** [review]
great job

File: server/controllers/ReviewsController.js (hunk 1 above)
Comment (stevekriz) [reply server/controllers/ReviewsController.js]: Agree. Updated.

File: server/index.js (hunk 2 above)
Comment (stevekriz) [reply server/index.js]: Updated.

File: server/routes.js (hunk 3 above)
Comment (stevekriz) [reply server/routes.js:4]: Thank you!

File: src/components/Reviews.jsx (hunk 4 above)
Comment (stevekriz) [reply src/components/Reviews.jsx:14]: Thank you!

File: webpack.config.js (hunk 5 above)
Comment (stevekriz) [reply webpack.config.js]: Updated.

**Review by oransproduce — APPROVED** [review]
wonderful!

**Pull request merged**